query-node/mappings/genesis.ts

```diff
@@ -30,6 +41,27 @@ export async function loadGenesisData({ store }: StoreContext): Promise<void> {
       )
     )
   )
+}
 
-  // TODO: members, workers
+async function initFirstElectionRound(store: DatabaseManager) {
+  const electedCouncil = new ElectedCouncil({
```

Comment:
There were actually two reasons:
1) I was not sure if there will be no council before the first election is complete or we will appoint a temporary council with preselected members (in that case, we could only add member ids into `genesis.ts`)
2) To remove checks for council existence in council mappings as well as in applications using QN

That being said, there is an inconsistency because there are a couple of places in `council.ts` that actually check that council record exists (as can be seen in your comment about `getCurrentElectedCouncil`/`getRequiredCurrentElectedCouncil`). 

I am for keeping the initial council in `genesis.ts` and removing all checks for elected council emptiness through `council.ts`. Do you think that's ok?

Reply:
After private discussion we've decided to keep initial council insertion in `genesis.ts`.